modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -191,6 +191,9 @@
   /** The key to look for in the service configuration file to override the {@link DEFAULT_INGEST_ZIP_JOB_LOAD} */
   public static final String ZIP_JOB_LOAD_KEY = "job.load.ingest.zip";
 
+  /** The default is to leave series catalogs untouched on ingest */
+  static final boolean DEFAULT_IS_OVERWRITE_SERIES = false;
```

Comment:
Ah, that constant, when said out loud, sounds a lot like it's going to overwrite the series ~ "The default is to overwrite the series (false)(??)". It might be better to just to put static final boolean FALSE  = false; :)

Reply:
```java
  static final boolean DEFAULT_ALLOW_OVERWRITE_SERIES = false;
 ```
?

Reply:
Ah, ok the "IS" version is more programer friendly, just not human friendly.

Reply:
or maybe 
```java
static final boolean DEFAULT_ALLOW_SERIES_WRITE = false;
```
since we are dealing with writes in general not necessarily only modifications but also series creation.

Reply:
Updated. Chose yet another name though :D